src/awala.ts

```diff
@@ -1,20 +1,33 @@
 import { getModelForClass } from '@typegoose/typegoose';
 
-import type { Result, SuccessfulResult } from './utilities/result.js';
+import type { Result } from './utilities/result.js';
 import type { ServiceOptions } from './serviceTypes.js';
 import type { MemberBundleRequest } from './schemas/awala.schema.js';
 import { MemberBundleRequestModelSchema } from './models/MemberBundleRequest.model.js';
+import { MemberPublicKeyModelSchema } from './models/MemberPublicKey.model.js';
 
 const contentTypeHeaderName = 'content-type';
 const awalaRecipientHeaderName = 'X-Awala-Recipient';
 
 export async function createMemberBundleRequest(
   requestData: MemberBundleRequest,
   options: ServiceOptions,
-): Promise<SuccessfulResult<undefined>> {
+): Promise<Result<undefined, undefined>> {
   const memberBundleRequestModel = getModelForClass(MemberBundleRequestModelSchema, {
     existingConnection: options.dbConnection,
   });
+  const memberPublicKeyModel = getModelForClass(MemberPublicKeyModelSchema, {
+    existingConnection: options.dbConnection,
+  });
+  const publicKey = await memberPublicKeyModel.findById(requestData.publicKeyId);
```

Comment:
We don't need the record, right? Let's do an _exists_ query

Reply:
actually, we need this to get the member Id
https://github.com/relaycorp/veraid-authority/pull/136/files#diff-725e18d8b08af438b9930029c9d9f1d74127beb38f17a39bc726315266bd435cR40